settings/docs/_index.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Settings"
+date: 2018-05-02T00:00:00+00:00
+weight: 10
+geekdocRepo: https://github.com/owncloud/ocis-settings
+geekdocEditPath: edit/master/docs
+geekdocFilePath: _index.md
+---
+
+## Abstract
+
+When using oCIS, the requirement to store settings arises. This extension provides functionality
+for other extensions to register new settings within oCIS. It is responsible for storing the respective
+settings values as well.
+
+For ease of use, this extension provides an ocis-web extension which allows users to change their settings values.
+Please refer to the [ocis-web extension docs](https://owncloud.github.io/ocis/extensions/#external-phoenix-apps)
+for running ocis-web extensions.
+
+{{< mermaid class="text-center">}}
+graph TD
+    subgraph ow[ocis-web]
+        ows[ocis-web-settings]
+        owc[ocis-web-core]
+    end
+    ows ---|"listSettingsBundles(),<br>saveSettingsValue(value)"| os[ocis-settings]
+    owc ---|"listSettingsValues()"| sdk[oC SDK]
+    sdk --- sdks{ocis-settings<br>available?}
+    sdks ---|"yes"| os
+    sdks ---|"no"| defaults[Use set of<br>default values]
```

Comment:
Codacy found an issue: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=6202733)